### PR TITLE
Updated examples/cli/README.md for directory encryption/decryption docs

### PR DESCRIPTION
**Pull request by SethLK**

### CODE REVIEW

File: examples/cli/README.md [hunk 1]
```diff
@@ -66,6 +66,24 @@ envcloak decrypt --input .env.enc --output .env --key-file mykey.key
 **Description:** Decrypts `.env.enc` back to `.env`. Ensure the `key-file` used matches the one from the encryption step.
 > ⚠️  Has additional `--force` flag to allow overwriting of decrypted files.
 
+### Encrypting Directories
+
+```bash
+envcloak encrypt --directory yourDirectory --output yourDirectory.enc --key-file mykey.key
+```
+
+**Description:** Encrypts your `yourDirectory` file into `yourDirectory.enc`. The original file remains unchanged.
```
Comment (Veinar) [inline examples/cli/README.md:75]: Almost, but not quite 😀 it encrypts directory and creates `output` directory (raises error if output directory exists, that's why `--force` flag is implemented). EnvCloak processes files one by one in input directory. 😉

File: examples/cli/README.md [hunk 2]
```diff
@@ -66,6 +66,24 @@ envcloak decrypt --input .env.enc --output .env --key-file mykey.key
...
+> ⚠️  Has additional `--force` flag to allow overwriting of encrypted directories.
+
+### Decrypting Directories
+
+```bash
+envcloak decrypt --directory yourDirectory.enc --output yourDirectory --key-file mykey.key
+```
+
+**Description:** Decrypts `yourDirectory.enc` back to `yourDirectory`. Ensure the `key-file` used matches the one from the encryption step.
```
Comment (Veinar) [inline examples/cli/README.md:84]: Same as above 😉

**Review by Veinar — CHANGES_REQUESTED** [review]

**Comment by Veinar** [comment]
Hi @SethLK, 👋

Thanks for contribution, great work 🙌, but please make changes requested and we'll be ready to go. 😉

File: examples/cli/README.md (hunk 2 above)
Comment (SethLK) [reply examples/cli/README.md]: How about now

**Review by Veinar — APPROVED** [review]

**Pull request merged**